### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by shinnn**
* I updated [findup-sync](https://github.com/cowboy/node-findup-sync), [multimatch](https://github.com/sindresorhus/multimatch) and [mocha](https://github.com/mochajs/mocha).
  * includes some important updates cowboy/node-findup-sync#14
* I added `sudo: false` to use [container-based infrastructure](http://docs.travis-ci.com/user/workers/container-based-infrastructure/) on Travis CI.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -18,7 +17,7 @@ module.exports = function(options) {
   options = options || {};
 
   var pattern = arrayify(options.pattern || ['gulp-*', 'gulp.*']);
-  var config = options.config || findup('package.json', {cwd: parentDir});
+  var config = options.config || require('findup-sync')('package.json', {cwd: parentDir});
```
Comment (jackfranklin) [inline index.js]: Could we still have the `var findup...` line up top? I like to have all my requires up top when possible :)

**Comment by jackfranklin** [comment]
Hey, sorry for the slow reply, I've been away. This is awesome, thank you so much. One small comment and then this is good to merge :)

**Comment by shinnn** [comment]
Updated.

**Comment by jackfranklin** [comment]
Thank you!

**Comment by jackfranklin** [comment]
Next time I make a change of any sort I'll do a full release :+1:

**Pull request merged**